packages/react/test/specs/lib/felaDisableAnimationsPlugin-test.ts

```diff
@@ -0,0 +1,60 @@
+import felaDisableAnimationsPlugin from 'src/lib/felaDisableAnimationsPlugin'
+
+const disableAnimationsPlugin = felaDisableAnimationsPlugin()
+
+describe('felaDisableAnimationsPlugin', () => {
+  test('does not disable animations if the props are not provided', () => {
+    const style = {
+      animation: 'k1',
+      margin: '0px 10px',
+    }
+
+    expect(disableAnimationsPlugin(style, 'RULE')).toMatchObject(style)
+  })
+
+  test('does not disable animations if the disableAnimations flag is undefined', () => {
+    const style = {
+      animation: 'k1 1s',
+      margin: '0px 10px',
+    }
+
+    expect(
+      disableAnimationsPlugin(style, 'RULE', undefined, { disableAnimations: undefined }),
+    ).toMatchObject(style)
+  })
+
+  test('does not disable animations if the disableAnimations flag is false', () => {
+    const style = {
+      animationName: 'k1',
+      animationDuration: '1s',
+      margin: '0px 10px',
+    }
+
+    expect(
+      disableAnimationsPlugin(style, 'RULE', undefined, { disableAnimations: false }),
+    ).toMatchObject(style)
+  })
+
+  test('disables animations if the disableAnimations flag is true', () => {
+    const style = {
+      animationName: 'k1',
+      animationDuration: '1s',
+      margin: '0px 10px',
+    }
```

Comment:
nit: We have the same object defined 5 times, can we move it out of `describe()` scope?

Reply:
Will restructure it a bit. We need few scenarios to test, but will make sure those are reused across the tests.

Reply:
Extracted two consts before the tests, which are used across all tests.